### ISSUE TITLE
Test failure: Interop/PInvoke/Vector2_3_4/Vector2_3_4/Vector2_3_4.sh

### ISSUE THREAD

**Issue by v-haren** [dotnet/runtime#37881]
failed in job: [runtime-coreclr jitstressregs 20200614.1 ](https://dev.azure.com/dnceng/public/_build/results?buildId=686944&view=ms.vss-test-web.build-test-results-tab&runId=21329030&resultId=100729&paneView=debug)

failed tests: 
Interop/PInvoke/Vector2_3_4/Vector2_3_4/Vector2_3_4.sh
Interop\\PInvoke\\Vector2_3_4\\Vector2_3_4\\Vector2_3_4.cmd

Error message
~~~
Return code: 1
Raw output file: /root/helix/work/workitem/Interop/PInvoke/Reports/Interop.PInvoke/Vector2_3_4/Vector2_3_4/Vector2_3_4.output.txt
Raw output:
BEGIN EXECUTION
/root/helix/work/correlation/corerun Vector2_3_4.dll ''
Running RunVector2Tests... 
Running RunVector3Tests... 
Running RunVector4Tests... 
TestLibrary.AssertTestException: Assert.AreEqual: Expected: [<42, 43, 274, 93761>]. Actual: [<93761, 999, 1203, 4>]. 
 at TestLibrary.Assert.HandleFail(String assertionName, String message) in /Users/runner/runners/2.170.1/work/1/s/src/coreclr/tests/src/Common/CoreCLRTestLibrary/Assertion.cs:line 735
 at TestLibrary.Assert.AreEqual[T](T expected, T actual, String message) in /Users/runner/runners/2.170.1/work/1/s/src/coreclr/tests/src/Common/CoreCLRTestLibrary/Assertion.cs:line 414
 at Vector2_3_4Test.RunVector4Tests() in /Users/runner/runners/2.170.1/work/1/s/src/coreclr/tests/src/Interop/PInvoke/Vector2_3_4/Vector2_3_4.cs:line 131
 at Vector2_3_4Test.Main() in /Users/runner/runners/2.170.1/work/1/s/src/coreclr/tests/src/Interop/PInvoke/Vector2_3_4/Vector2_3_4.cs:line 20
Expected: 100
Actual: 101
END EXECUTION - FAILED
Test Harness Exitcode is : 1
To run the test:
> set CORE_ROOT=/root/helix/work/correlation
> /root/helix/work/workitem/Interop/PInvoke/Vector2_3_4/Vector2_3_4/Vector2_3_4.sh
Expected: True
Actual: False


Stack trace
   at Interop_PInvoke._Vector2_3_4_Vector2_3_4_Vector2_3_4_._Vector2_3_4_Vector2_3_4_Vector2_3_4_sh() in /__w/1/s/artifacts/tests/coreclr/Linux.arm64.Checked/TestWrappers/Interop.PInvoke/Interop.PInvoke.XUnitWrapper.cs:line 2759
~~~

**Comment by BruceForstall** [comment]
This is arm64, Windows and Linux, with
```
set COMPlus_TieredCompilation=0
set COMPlus_JitStressRegs=8
```

**Comment by CarolEidt** [comment]
I discovered this last week and it is now fixed with #37773
